NuGet.config

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
+    <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
+    <!-- Required when building within dotnet/aspnetcore. -->
+    <add key="dotnet6-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6-transport/nuget/v3/index.json" />
```

Comment:
I suspect this is required in dotnet/aspnetcore builds because we use the Microsoft.AspNetCore.Internal.Transport package almost everywhere. Decided this was minor as long as the feed didn't impact local builds (it doesn't).